doc/libfuzzer.md

```diff
@@ -0,0 +1,57 @@
+# libfuzzer
+
+## setup 
+
+### Requirements
+libFuzzer needs LLVM sanitizer support, 
+so this only works on x86-64 Linux, 
+x86-64 macOS and Apple-Silicon (aarch64) macOS for now. 
+This also needs a nightly compiler since 
+it uses some unstable command-line flags. 
+You'll also need a C++ compiler with C++11 support.
+
+
+
+### Installing
+
+rust-version|libfuzzer|td-shim|result
+:-:|:-:|:-:|:-:
+nightly-2020-02-08|0.11.0|staging|no (td-shim fails to compile in this version)
+nightly-2021-08-20|0.10.2|staging|yes
+nightly-2020-02-08|0.11.0|staging|no (libfuzzer failed to start)
```

Comment:
- These are all very old builds now. Is there still a problem building with newer versions?
- What libfuzzer error do you get with 0.11.0? Is there an open issue on that project for the problem?

Reply:
1. I wrote the wrong version number, it has been revised correctly.
2. I reported an issue in cargo-fuzz.